src/components/Pages/SetupPage/SetupPage.tsx

```diff
@@ -69,61 +202,110 @@ export const SetupPage = () => {
-                placeholder="Enter your email"
-                message={
-                  <>
-                    {showErrors && emailError ? (
-                      <>
-                        <span className={styles.emailError}>{emailError}</span>
-                        <br />
-                      </>
-                    ) : null}
-                    <span>Subscribe to Decentraland's newsletter to receive the latest news about events, updates, contests and more.</span>
-                  </>
-                }
-                onChange={onEmailChange}
-              />
+  if (view === View.RANDOMIZE) {
+    return (
+      <div className={styles.container}>
+        <div className={styles.background} />
+        <div className={styles.left}>
+          <div className={styles.leftInner}>
+            <img className={styles.logo} src={logoImg} alt="logo" />
+            <div className={styles.title}>Welcome to Decentraland!</div>
+            <div className={styles.subtitle}>Your journey begins here</div>
+            <div className={styles.meetYourAvatar}>First, Meet Your Avatar</div>
+            <div className={styles.meetYourAvatarDescription}>
+              This is your new digital self!
+              <br />
+              Don't worry if it's not quite 'you' yet - you'll later have plenty of options to make it your own.
             </div>
-            <div className={styles.agree}>
-              <Checkbox className={styles.checkbox} onChange={onAgreeChange} />
-              &nbsp;I agree with Decentraland's&nbsp;
-              <a target="_blank" rel="noopener noreferrer" href="https://decentraland.org/terms/">
-                Terms of use
-              </a>
-              &nbsp;and&nbsp;
-              <a target="_blank" rel="noopener noreferrer" href="https://decentraland.org/privacy">
-                Privacy policy
-              </a>
+            <div className={styles.randomize}>
+              <Button compact inverted onClick={onRandomize}>
+                <img src={diceImg} alt="diceImg" />
+                <span>randomize</span>
+              </Button>
             </div>
-            {showErrors && agreeError ? <div className={styles.agreeError}>{agreeError}</div> : null}
-            <div className={styles.jumpIn}>
-              <Button primary fluid type="submit">
-                Jump in decentraland
+            <div className={styles.continue}>
+              <Button primary fluid onClick={onContinue}>
+                continue
               </Button>
             </div>
-          </form>
+          </div>
+        </div>
+        <div className={styles.right}>
+          <WearablePreview lockBeta={true} panning={false} disableBackground={true} profile={profile} dev={false} />
+          <img className={styles.platform} src={platformImg} alt="platform" />
         </div>
       </div>
-      <div className={styles.right}></div>
-    </div>
-  )
+    )
+  }
+
+  if (view === View.FORM) {
```

Comment:
What do you think about rendering this view as default instead of putting it under a condition and rendering null at the bottom?

Reply:
Done